lib/moesif_rack/app_config.rb

```diff
@@ -3,14 +3,14 @@
 require 'time'
 require 'zlib'
 require 'stringio'
-require_relative './helpers.rb'
+require_relative './moesif_helpers.rb'
 require_relative './regex_config_helper.rb'
 
 class AppConfig
 
     def initialize debug
         @debug = debug
-        @helpers = Helpers.new(debug)
+        @helpers = MoesifHelpers.new(debug)
```

Comment:
I'd suggest to change the instance variable to `moesif_helpers` as well to avoid any issue.